.github/actions/send-slack-notification/action.yml

```diff
@@ -81,6 +81,26 @@ runs:
         fi
         echo "result=$RESULT" >> $GITHUB_OUTPUT
 
+    - name: Compute run link
+      id: compute-run-link
+      shell: bash
+      env:
+        REPO_URL: ${{ github.server_url }}/${{ github.repository }}
+        WORKFLOW: ${{ github.workflow }}
+      run: |
+        RESULT="$REPO_URL/actions/runs/$GITHUB_RUN_ID/attempts/$GITHUB_RUN_ATTEMPT"
+        echo "result=$RESULT" >> $GITHUB_OUTPUT
+
+    - name: Compute workflow title
+      id: compute-workflow-title
+      shell: bash
+      run: |
+        RESULT="$GITHUB_WORKFLOW"
+        if [[ "$GITHUB_RUN_ATTEMPT" != '1' ]]; then
+          RESULT+=" (attempt #$GITHUB_RUN_ATTEMPT)"
+        fi
```

Comment:
that means that the first run attempt will always link to the last attempt, right?

Reply:
No the URL will always have attempt/1, here it's just the title that does not state "(attempt 1)" after the workflow title